src/workers/notifications/commentReply.ts

```diff
@@ -37,30 +37,24 @@ const worker: NotificationWorker = {
     if (!postCtx) {
       return;
     }
-    const [parent, commenter, mutes] = await Promise.all([
+    const [parent, commenter] = await Promise.all([
       comment.parent,
       comment.user,
-      con.getRepository(NotificationPreferenceComment).findBy({
-        referenceId: comment.id,
-        notificationType: In(commentReplyNotificationTypes),
-      }),
     ]);
-    const threadFollowers = await con
-      .getRepository(Comment)
-      .createQueryBuilder()
-      .select('"userId"')
-      .distinct(true)
-      .where('"parentId" = :parentId', { parentId: parent.id })
-      .andWhere('"userId" not in (:...exclude)', {
-        exclude: [parent.userId, comment.userId],
-      })
-      .getRawMany();
```

Comment:
Where did this go?

Reply:
This is point a, its spammy to receive these if you simply in the thread 👍

Reply:
Ah right!